### ISSUE TITLE
Please show the red edges and rectangles for the current query without requiring the user to click on its node.

### ISSUE THREAD

**Issue by amichail** [anvaka/sayit#2]
Also, please don't show a preview of the subreddit when the user clicks on a node.

**Comment by anvaka** [comment]
why not show the preview?

**Comment by amichail** [comment]
> why not show the preview?

I guess it’s ok to show the preview if you show the initial graph with red edges emanating from the query node. I was constantly selecting the query node just to see the red edges, but this obscured the graph with the preview.

**Comment by amichail** [comment]
> Please check now - is this what you are expecting?

Yes, but perhaps these additional changes would help: (1) visually distinguish the current query node from the other red nodes somehow; (2) only show the preview when you (single) click on the current query node.
